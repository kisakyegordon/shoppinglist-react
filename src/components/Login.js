import React, { Component } from 'react';
import LoginForm from '../components/forms/loginform';

class Login extends Component{

    render (){
        return (
            <div>
            <h2> Log in </h2>
            <LoginForm />
            </div>
        );
    }

}

export default Login;