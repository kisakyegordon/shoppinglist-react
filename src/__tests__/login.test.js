import React from 'react';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import Login from '../components/login/login';

describe('<Login />', () => {
    it('match a snapshot', () => {
        const wrapper = shallow(<Login />);
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });

});