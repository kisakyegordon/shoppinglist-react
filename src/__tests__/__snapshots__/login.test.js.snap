// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> match a snapshot 1`] = `
<div>
  <ToastContainer
    autoClose={5000}
    bodyClassName=""
    className={null}
    closeButton={<DefaultCloseButton />}
    closeOnClick={true}
    hideProgressBar={true}
    newestOnTop={false}
    pauseOnHover={true}
    position="top-right"
    progressClassName=""
    style={null}
    toastClassName=""
    transition={[Function]}
  />
  <div
    className="signin"
  >
    <h1>
       Log in 
    </h1>
    <LoginForm />
  </div>
</div>
`;
